qa.md

```diff
@@ -62,21 +205,122 @@
 }
 ```
 
+## TSM009: Try to send an email with no template
 
-## TSM03: Try to send an email with missing template
+`PUT /send_mail`
+
+```
+{
+    "to": ["foo@example.com"],
+    "body": {
+    }
+}
+```
+
+## TSM010: Try to send an email with base64 attachment.
 
 `PUT /send_mail`
 
 ```
 {
     "to": ["foo@example.com"],
     "body": {
-        "template": "/Templates/hello.html"
+        "template": "/Templates/Emails/hello.html"
     },
-    'attachments': [{
-        'base64': '',
-        'content-type': 'text/csv',
-        'filename': 'c7fbbb3d716d4d7c95d3b887b288ed62.csv'
+    "attachments": [{
+        "base64": "",
```

Comment:
There should be something here, small Base64 CSV

Reply:
@ateska Can I merge this?

Reply:
Can you actually include the base64 encoded small CSV file here?

Reply:
@ateska It was here.

https://github.com/TeskaLabs/asab-iris/pull/14/files#diff-6ffbbeade89893ca3b4a89dc9c06569fc1b77827c669f0493a92e1c2c716e152R253

Reply:
OK